index.html

```diff
@@ -973,7 +978,7 @@ <h3 property="schema:name">Authorization Extensions</h3>
                 <div datatype="rdf:HTML" property="schema:description">
                   <p><em>This section is non-normative.</em></p>
 
-                  <p>As ACL resources are RDF sources, <a href="#authorization">Authorizations</a> descriptions may be extended or limited by constraints eg. temporal or spatial constraints, and duties eg. payments, may be imposed on permissions, but no behaviour is defined by this specification. For example, the <cite><a href="https://www.w3.org/TR/odrl-model/" rel="cito:citesAsPotentialSolution">ODRL Information Model</a></cite> can be used to set obligations required to be met by agents prior to accessing a resource.</p>
+                  <p>As ACL resources are RDF sources, <a href="#authorization">Authorizations</a> descriptions may be extended or limited by constraints, eg. temporal or spatial constraints, and duties, eg. payments, may be imposed on permissions, but no behaviour is defined by this specification. For example, the <cite><a href="https://www.w3.org/TR/odrl-model/" rel="cito:citesAsPotentialSolution">ODRL Information Model</a></cite> can be used to set obligations required to be met by agents prior to accessing a resource.</p>
```

Comment:
```suggestion
                  <p>As ACL resources are RDF sources; <a href="#authorization">Authorization</a> descriptions may be extended or limited by constraints, e.g., temporal or spatial constraints; and duties, e.g., payments, may be imposed on permissions; but no behaviour is defined by this specification. For example, the <cite><a href="https://www.w3.org/TR/odrl-model/" rel="cito:citesAsPotentialSolution">ODRL Information Model</a></cite> can be used to set obligations required to be met by agents prior to accessing a resource.</p>
```
Solid should decide whether its documents (including this one) should use UK or US English spellings, particularly with "o" vs "ou" (such as "behaviour", here), and "z" vs "s" (such as "organised", among others, above)

Reply:
Right, given `acl:Authorization` it seems we are most inclined to use US English spellings...

Reply:
As mentioned in https://github.com/solid/web-access-control-spec/pull/83#issuecomment-865356721 , "authorization" in BrE is fine. See e.g. https://english.stackexchange.com/questions/282621/authorization-vs-authorisation-im-in-some-real-dilemma . If that's not reasonable, we can also battle this in HTML e.g.:

`<html lang="en"> ... <span lang="en-us">authorization</span>` :)

Reply:
WFM! :-)